### PR TITLE
ND-4243 Update portal settings api

### PR DESCRIPTION
**Pull request by katerina-zaitsava**

### CODE REVIEW

File: src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts [hunk 1]
```diff
@@ -3,11 +3,16 @@ import { BorrowerType } from '../../enums';
 import DigitalLendingPortalEnvironment from '../../enums/DigitalLendingPortalEnvironment';
 import getSearchParams from '../../utils/getSearchParams';
 
-export interface BorrowerStandardPortalGeneralSettings {
+export interface StandardBorrowerPortalConfig {
```
Comment (ilya-tishonok) [inline src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts:6]: is this interface was implemented in prior version? (not in alpha)

Comment (katerina-zaitsava) [reply src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts:6]: no, it's a new one

Comment (ilya-tishonok) [reply src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts:6]: i mean, BorrowerStandardPortalGeneralSettings

Comment (katerina-zaitsava) [reply src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts:6]: BorrowerStandardPortalGeneralSettings existed before

Comment (katerina-zaitsava) [reply src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts:6]: but with different properties

**Review by ilya-tishonok — APPROVED** [review]

**Pull request merged**